.php_cs.dist

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * DO NOT EDIT THIS FILE!
+ *
+ * It's auto-generated by sonata-project/dev-kit package.
+ */
+
+$header = <<<'HEADER'
+This file is part of the Sonata Project package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
```

Comment:
This is okay? The original package was not created by thomas.

Reply:
For what I understand, it means that it's inside the `sonata-project` organisation, which is true now.